### PR TITLE
Misuv 7025 no encryption boolean

### PR DESCRIPTION
**Pull request by ayubAz**

### CODE REVIEW

**Review by avelx — APPROVED** [review]
approved with comment

File: app/services/SessionService.scala [hunk 1]
```diff
@@ -81,7 +94,10 @@ class SessionService @Inject()(uiJourneySessionDataRepository: UIJourneySessionD
 
   def setMongoData(uiJourneySessionData: UIJourneySessionData)
                   (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Boolean] = {
-    uiJourneySessionDataRepository.set(uiJourneySessionData)
+    if (config.encryptionIsEnabled)
```
[the remark this answers is not in the document]
Comment (avelx) [reply app/services/SessionService.scala:97]: the problem is that we never return false from this method, so use of Boolean type redundant

File: conf/application.conf [hunk 2]
```diff
@@ -294,3 +294,8 @@ bootstrap.http.headersAllowlist = [ "Gov-Test-Scenario", "Location", "X-Request-
 internalServiceHostPatterns = [ "localhost" ]
 
 accessibility-statement.service-path = "/income-tax-view-change"
+
+encryption {
+  key = "QmFyMTIzNDVCYXIxMjM0NQ=="
+  isEnabled = false
+}
```
Comment (EdwardDennis) [inline conf/application.conf:301]: I think this should be stored in app-config-base instead
See here, encryption not to be stored in application.conf
https://github.com/hmrc/app-config-base/blob/main/documentation/README.md#applicationconf

Comment (EdwardDennis) [reply conf/application.conf:301]: This key should be changed as well, given that this repo is public

**Pull request merged**